package.json

```diff
@@ -128,5 +127,11 @@
     "make-error": "^1.1.1",
     "source-map-support": "^0.5.17",
     "yn": "3.1.1"
+  },
+  "prettier": {
+    "singleQuote": true,
+    "trailingComma": "all",
+    "printWidth": 120,
+		"useTabs": true
```

Comment:
Why not just use prettier with defaults completely?

Reply:
I'm not overly attached to any choices; happy to change them.

tabs seem like the better choice for accessibility by the visually impaired.  At least as far as I can read online (https://github.com/prettier/prettier/issues/7475, https://www.reddit.com/r/javascript/comments/c8drjo/nobody_talks_about_the_real_reason_to_use_tabs/)

singleQuote is to match existing style but it's not a big deal

No preference on printWidth.  I had assumed without evidence that the default would cause a lot of lines to be wrapped.  I tested and very few were actually affected